---
description: '자세한 정보: 컴파일러 오류 CS1930'
title: 컴파일러 오류 CS1930
ms.date: 07/20/2015
f1_keywords:
- CS1930
helpviewer_keywords:
- CS1930
ms.assetid: d28d2334-825c-4ffc-b9e9-f5d61f44d672
ms.openlocfilehash: 84f90ce9ba4b145a00b511ba53dc9be7109ec774
ms.sourcegitcommit: ddf7edb67715a5b9a45e3dd44536dabc153c1de0
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 02/06/2021
ms.locfileid: "99730202"
---
# <a name="compiler-error-cs1930"></a>컴파일러 오류 CS1930

'name' 범위 변수가 이미 선언되었습니다.  
  
 쿼리 식의 범위 변수는 쿼리 식이 종료될 때까지 범위 내에 있습니다. 따라서 고유 식별자가 있어야 합니다.  
  
## <a name="to-correct-this-error"></a>이 오류를 해결하려면  
  
1. 쿼리 식에 사용된 각 범위 변수에 고유한 이름을 지정합니다.  
  
## <a name="example"></a>예제  

 다음 예제에서는 `num` 절의 범위 변수와 `from` 절에 사용된 범위 변수에 대해 `let` 식별자가 사용되기 때문에 CS1930을 생성합니다.  
  
```csharp  
// cs1930.cs  
using System.Linq;  
class Program  
{  
    static void Main()  
    {  
        int[] nums = { 0, 1, 2, 3, 4, 5 };  
        var query = from num in nums  
                    let num = 3 // CS1930  
                    select num;
    }  
}  
```  
  
## <a name="see-also"></a>참조

- [LINQ 쿼리 식](../linq/index.md)
